src/containers/SearchPage/SearchPage.helpers.js

```diff
@@ -13,11 +13,12 @@ import routeConfiguration from '../../routeConfiguration';
  * @param {Object} paramValue Search parameter value
  * @param {Object} filters Filters configuration
  */
-export const validURLParamForExtendedData = (paramName, paramValue, filters) => {
+export const validURLParamForExtendedData = (paramName, paramValueRaw, filters) => {
   const filtersArray = Object.values(filters);
   // resolve configuration for this filter
   const filterConfig = filtersArray.find(f => f.paramName === paramName);
 
+  const paramValue = paramValueRaw.toString();
```

Comment:
I added this because if the value in keyword search is e.g. "123" the app would crash otherwise as the value is a number and not a string and it doesn't have split() function.